impl/src/main/java/org/ehcache/impl/serialization/CompactJavaSerializer.java

```diff
@@ -92,12 +93,12 @@ public CompactJavaSerializer(ClassLoader loader) {
 
   @Override
   public void init(final StateRepository stateRepository) {
-    this.readLookup = stateRepository.getPersistentConcurrentMap("CompactJavaSerializer-ObjectStreamClassIndex", Integer.class, ObjectStreamClass.class);
+    this.readLookup = stateRepository.getPersistentStateHolder("CompactJavaSerializer-ObjectStreamClassIndex", Integer.class, ObjectStreamClass.class);
     loadMappingsInWriteContext(readLookup.entrySet(), true);
   }
 
   Map<Integer, ObjectStreamClass> getSerializationMappings() {
-    return Collections.unmodifiableMap(new HashMap<Integer, ObjectStreamClass>(readLookup));
+    return Collections.unmodifiableMap(new HashMap<Integer, ObjectStreamClass>(Collections.<Integer, ObjectStreamClass>emptyMap()));
```

Comment:
Ignore this change as this method has been removed in https://github.com/ehcache/ehcache3/pull/1435